server/audio/natural_language_processing.py

```diff
@@ -89,16 +89,21 @@ def authenticate():
 
 
 def analyze_action(text):
-    actions = []
     service = authenticate()
-    response = service.analyze(
-        text=text,
-        features=Features(semantic_roles=SemanticRolesOptions())).get_result()
-    for resp in response['semantic_roles']:
-        actions.append(resp['action']['text'])
-    if len(actions) == 0:
-        raise ActionNotFoundException
-    return actions
+    try:
+        response = service.analyze(
+            text=text,
+            features=Features(semantic_roles=SemanticRolesOptions())).get_result()
+        raw_resp = response['semantic_roles'][-1]
+        action = raw_resp['action']['text']
+        try:
+            action += ' ' + raw_resp['object']['text']
+        except ApiException:
+            # It may or may not have an object
+            pass
```

Comment:
Aca si no existe object no es mejor poner esa condicion? onda: `if 'object' in raw_resp: ...`